src/v2/Components/v2/ArtworkFilter/index.tsx

```diff
@@ -92,7 +92,7 @@ export const BaseArtworkFilter: React.FC<
       | ArtistSeriesArtworksFilter_artistSeries
       | FairArtworks_fair
       | ShowArtworks_show
-    Filters?: React.FC
+    Filters?: JSX.Element
```

Comment:
I was able to solve the collapse-on-selection problem by having the show and fair artworks directly pass the filters that they wanted to use, instead of passing a function that return those filters. The functional component was being called each time the filter state changed, and I believe that this was causing the filters to revert to their default, collapsed state.